### DISCUSSION TITLE
v4.0-beta.2 pre-release

### DISCUSSION THREAD

**Discussion by stickz** [Novik/ruTorrent#2315] [General]
Hi Everyone,

I'm currently working on getting anther pre-released tagged for ruTorrent. @liaralabs needs this for [swizzin](https://github.com/swizzin/swizzin) to unpin the [file manager](https://github.com/nelu/rutorrent-filemanager) version. Many bugs have also been fixed since the last version tag. A stack trace for javascript errors was also added. 

Special thanks goes to @TrimmingFool for fragmenting the rXMLRPC calls and improving the WebUI. We can perform operations on thousands of torrents now without errors. This has been a long time issue with ruTorrent, which has finally been resolved.

**Would anyone like to translate the language files on master?**
https://github.com/Novik/ruTorrent/tree/master/lang

It would be great to distribute at least a few completed languages with this tag. @Micdu70 @Novik @ChriZathens @babico

**Comment by TrimmingFool** [comment]
The DarkBetter theme is missing one commit. 
https://github.com/chocolatkey/DarkBetter/pull/4
@chocolatkey

**Comment by chocolatkey** [comment]
@TrimmingFool fixed

**Comment by ChriZathens** [comment]
Hello @stickz !
Regarding translating: I think all is translated (at least in Greek 🇬🇷  that I translate)
Or am I missing something? (I followed your link above https://github.com/Novik/ruTorrent/tree/master/lang )
Let me know if I am ovelooking something, please! 😃

**Reply by stickz** [reply]
Hi @ChriZathens. I have anther link for the plugins. Anything that was updated in the last month may need to be translated.
https://github.com/Novik/ruTorrent/tree/master/plugins

**Comment by stickz** [comment]
@liaralabs sorry for the delay. My computer broke. [ruTorrent v4.0-beta.2](https://github.com/Novik/ruTorrent/releases/tag/v4.0-beta.2) has been released! It is very stable and bug free. 

It's recommended to set `$do_diagnostic = false;` in `/conf/config.php` when unpinning file manager. The developer implemented an auto loader in the plugin. We downgraded the errors to warnings. And suppressed the warnings with this variable set to false. We have no plans to fully support file manager in diagnostic mode. It would add to much overhead to a setting we want to keep enabled by default.

It would probably be better for your project anyways to disable diagnostic mode. Swizzin sets up the whole seed box environment itself. This is less prone to configuration issues. Any issues can be detected during the installation process instead. This will make ruTorrent run faster!

**Reply by liaralabs** [reply]
Thanks for the update! We've got a commit backlog in develop right now so maybe I'll use this opportunity to unpin FM and merge a new release into the master branch this weekend
